<ui version="4.0" >
 <class>CmdBindingConfigWidget</class>
 <widget class="QWidget" name="CmdBindingConfigWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>335</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTreeWidget" name="treeWidget" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <column>
      <property name="text" >
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Command</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Description</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnEditEntry" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Edit entry...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemoveEntry" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Remove entry</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>201</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddEntry" >
       <property name="text" >
        <string>Add entry...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gbNotes" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>3</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Further Notes</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="lblNotes" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>3</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>&lt;p>The entries are accessible through the submenu &lt;b>Commands&lt;/b> in the &lt;b>Tools&lt;/b> menu. For faster access it is possible to assign &lt;b>shortcuts&lt;/b> in the shortcut configuration page after applying the changes.&lt;/p></string>
        </property>
        <property name="textFormat" >
         <enum>Qt::RichText</enum>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="lblIcon" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>1</hsizetype>
            <vsizetype>1</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>31</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="KSeparator" name="kseparator" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KSeparator</class>
   <extends></extends>
   <header>kseparator.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>treeWidget</tabstop>
  <tabstop>btnAddEntry</tabstop>
  <tabstop>btnRemoveEntry</tabstop>
  <tabstop>btnEditEntry</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
